snap/hooks/configure

```diff
@@ -385,6 +385,13 @@ then
   need_api_restart=true
 fi
 
+# Disable the watchlist feature until k8s-dqlite can handle it
+if ! grep -e "feature-gates=WatchList=false" ${SNAP_DATA}/args/kube-apiserver
```

Comment:
is this enough? What if some one has set `feature-gates=AnotherFeature=True,WatchList=false`? Or what if someone has `WatchList=true` but is running etcd?

I think we should detect the absence of `WatchList` and make sure it is set to `false`.